include/dblayer/mysql.php

```diff
@@ -182,7 +182,7 @@ function close()
 	{
 		if ($this->link_id)
 		{
-			if ($this->query_result)
+			if (is_object($this->query_result))
```

Comment:
@Quy That is the mysqli_ function documentation. The mysql_ functions return a resource as describe [here](http://php.net/manual/en/function.mysql-query.php#refsect1-function.mysql-query-returnvalues).